singer-commons/pom.xml

```diff
@@ -54,6 +54,26 @@
       <artifactId>commons-configuration</artifactId>
       <version>1.9</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka_2.11</artifactId>
+      <version>2.1.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>2.1.0</version>
```

Comment:
Refactor version to property in pom.xml ?

Reply:
changed together with the Kafka dependency